src/config_rules.h

```diff
@@ -66,6 +66,7 @@ struct SacrificeRecipe {
 /******************************************************************************/
 extern const char keeper_rules_file[];
 extern const struct NamedCommand research_desc[];
+extern const struct NamedCommand rules_game_classicbugs_commands[];
```

Comment:
What does this do? Has it anything to do with the console commands?

Reply:
Yes, the bug toggling command needs access to it.